packages/zowe-explorer/src/globals.ts

```diff
@@ -30,6 +30,8 @@ export let DS_DIR;
 export let ISTHEIA: boolean = false; // set during activate
 export let LOG: Logger;
 export const COMMAND_COUNT = 93;
+export const MAX_SEARCH_HISTORY = 5;
```

Comment:
Do we need to specify, that these constants are used only in 'uss' tree? Or maybe we can rename/replace them somewhere?

Reply:
They aren't just used in USS tree, they are used for any list of previous searches (also DS and jobs trees have search capability)

And MAX_FILE_HISTORY is also in use for DS tree, you can see recently-opened data sets & members.

Reply:
Hmm, this is weird, I thought you just add these constants.. okay :)